test/unit/models/forms/DecisionFormTest.scala

```diff
@@ -100,15 +105,15 @@ class DecisionFormTest extends ModelsBaseSpec {
         val form                          = formProvider(mockedCommodityCodeConstraint).liabilityCompleteForm(decision)
 
         form.hasErrors shouldBe false
-        form.data      shouldBe params.mapValues(v => v.head)
+        form.data      shouldBe params.map { case (fst, snd) => fst -> snd.head }
```

Comment:
Same with the others as it can just be `params.view.mapValues(v => v.head).toMap`. Thanks